pkg/util/filesystem/util_windows.go

```diff
@@ -260,3 +261,79 @@ func Chmod(path string, filemode os.FileMode) error {
 func IsAbs(path string) bool {
 	return filepath.IsAbs(path) || strings.HasPrefix(path, `\`) || strings.HasPrefix(path, `/`)
 }
+
```

Comment:
could you write a ut for these two new functions?

Reply:
Sure.